base/errorshow.jl

```diff
@@ -892,3 +893,14 @@ function show(io::IO, ip::InterpreterIP)
         print(io, " in $(ip.code) at statement $(Int(ip.stmt))")
     end
 end
+
+# handler for displaying a hint in case the user tries to call
+# the instance of a number(misses out hte operator)
+# eg: (1 + 2)(3 + 4)
+function sym_hint_handler(io, ex, arg_types, kwargs)
```

Comment:
Any particular reason to call this `sym_hint_handler`? I would probably have called it something like `noncallable_number_hint_handler`, just to make it a little bit clearer.

Reply:
No particular reason I guess.